app/controllers/professors_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ProfessorsController < ApplicationController
+
+  def index
+    @professors = Professor.all.order(:name)
```

Comment:
I like your inclination not to use the order method in the view, however, this shouldn't live in the controller either. Convention is usually to create a method in the model that you would call in the controller. For example, if you have a method called `order_by_name` in your model, your controller then would like `@professors = Professor.order_by_name`